### PR TITLE
[RTM] PHP Toolbox support

### PR DESCRIPTION
**Pull request by aschempp**
The [PHP Toolbox](https://plugins.jetbrains.com/plugin/8133) plugin adds customizable autocomplete additions to PhpStorm. Here's a simple draft for Contao `core-bundle`.
- Autocomplete and type detection of `ContaoFrameworkInterface::getAdapter()`
  
- Autocomplete for DCA and languages (currently hardcoded)
  
- Autocomplete for model options
  

These are just examples and if we come up with more ideas it's highly extendable 😎

### CODE REVIEW

**Comment by leofeyer** [comment]
Nice. :v:

**Comment by aschempp** [comment]
Is this something we want to add to the core, event though it's IDE specific?

If so, let's think of other places we could use autocomplete. And no, DCA probably wont work 😂

**Comment by aschempp** [comment]
I just thought about this again, and I can't come up with additional ideas. I think this can be merged as-is, and we'll simply extend when we find something.

File: .ide-toolbox.metadata.json [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+                "tl_templates",
+                "tl_theme",
+                "tl_undo",
+                "tl_user",
+                "tl_user_group"
+            ]
+        },
+        {
+            "name": "contao_languagefiles",
+            "lookup_strings": ["default", "explain"]
```
Comment (leofeyer) [inline .ide-toolbox.metadata.json]: Should we add the other core-bundle language files here?

File: .ide-toolbox.metadata.json [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+                    "type_text": "integer"
+                },
+                {
+                    "lookup_string": "offset",
+                    "type_text": "integer"
+                },
+                {
+                    "lookup_string": "return",
+                    "type_text": "Model|Collection|Array"
+                }
```
Comment (leofeyer) [inline .ide-toolbox.metadata.json]: Here are the options I have found:

 * [ ] `table`
 * [x] `column`
 * [x] `value`
 * [x] `limit`
 * [x] `offset`
 * [x] `return` 
 * [ ] `order` (probably what you mean by `sorting`)

File: .ide-toolbox.metadata.json [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+                    "method": "getAdapter",
+                    "type": "type"
+                }
+            ]
+        },
+        {
+            "provider":"contao_framework_classes",
+            "language":"php",
+            "signature":[
+                "Contao\\CoreBundle\\Framework\\ContaoFrameworkInterface:getAdapter"
```
Comment (aschempp) [inline .ide-toolbox.metadata.json]: The provider is different. The first one allows subclasses of `Contao\System`, but some are not (like `System` itself and `Config`, maybe there's even more).

File: .ide-toolbox.metadata.json (hunk 1 above)
Comment (aschempp) [inline .ide-toolbox.metadata.json]: It would not harm, I was just too lazy in the draft :-)

File: .ide-toolbox.metadata.json (hunk 2 above)
Comment (aschempp) [inline .ide-toolbox.metadata.json]: Fixed in c8ecd792f9afa9b4795a171950921bbeb0a51229

**Pull request merged**